examples/02-javascript-basics/helpers/array_equality.js

```diff
@@ -0,0 +1,19 @@
+Array.isEqual = function (array1, array2) {
+    if (!array1 || !array2)
```

Comment:
@paulomilitante Good catch for null/undefined values. However, this will also fail for:
```
Array.isEqual([], []) // false
```

Which should be true. Kindly update to take care of this scenario.